### PR TITLE
Add new PostGIS Sink for cygnus-ngsi-ld agent

### PR DESCRIPTION
**Pull request by anmunoz**
New PostGIS Sink fro working with NGSI-LD notification using cygnus-ngsi-ld agent, this development is part of the collaboration between the UPM and FIWARE foundation. We provide also the tests and documentation examples. This sink is capable to save into a PostGIS using the geometry properties of PostgreSQL in the same way as the PostgisSink for cygnus-ngsi. The provided Sink is capable to process the following type of GeoProperties:

Point
Multipoint
LineString
MultiLineString
Polygon

### CODE REVIEW

**Comment by fgalan** [comment]
Thank you for your contribution!

Please add an entry in CHANGES_NEXT_RELEASE with a brief description (one sentence) of the changes introduced in this PR, in the following format:

```
[cygnus-ngsi-ld] <description>
```

**Comment by fgalan** [comment]
In addition, we have now a problem with RTD documentation generation. The RTD built fails at this point:

```
ERROR   -  File not found: /home/docs/checkouts/readthedocs.org/user_builds/fiware-cygnus/checkouts/master/doc/cygnus-ngsi-ld/user_and_programmer_guide/programmer_guide.md 
ERROR   -  Error building page cygnus-ngsi-ld/user_and_programmer_guide/programmer_guide.md 
```

Probably this was caused by PR https://github.com/telefonicaid/fiware-cygnus/pull/1885 but we didn't realize.

Could you have a look and ensure that all documentation cross-references are ok? Maybe mkdocs.yml is citing a file that is not found...

File: doc/cygnus-ngsi-ld/flume_extensions_catalogue/ngsi_postgis_sink.md [hunk 1]
```diff
@@ -0,0 +1,169 @@
+# <a name="top"></a>NGSIPostGISSink
```
Comment (fgalan) [inline doc/cygnus-ngsi-ld/flume_extensions_catalogue/ngsi_postgis_sink.md:1]: This .md file should be added to mkdocs.yml so it can be rendered by RTD.

Comment (anmunoz) [reply doc/cygnus-ngsi-ld/flume_extensions_catalogue/ngsi_postgis_sink.md:1]: Ok, I included it in the documentation

**Review by fgalan — APPROVED** [review]
LGTM

**Pull request merged**